server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/nano/test/TestPaymentSupport.java

```diff
@@ -0,0 +1,79 @@
+package com.generalbytes.batm.server.extensions.extra.nano.test;
+
+import com.generalbytes.batm.server.extensions.extra.nano.NanoExtension;
+import com.generalbytes.batm.server.extensions.extra.nano.NanoPaymentSupport;
+import com.generalbytes.batm.server.extensions.extra.nano.NanoCurrencySpecification;
+import com.generalbytes.batm.server.extensions.extra.nano.wallets.node.NanoRPCClient;
+import com.generalbytes.batm.server.extensions.extra.nano.wallets.node.NanoNodeWallet;
+import com.generalbytes.batm.server.extensions.extra.nano.wallets.node.NanoWSClient;
+import com.generalbytes.batm.server.extensions.payment.IPaymentRequestListener;
+import com.generalbytes.batm.server.extensions.payment.PaymentRequest;
+import uk.oczadly.karl.jnano.rpc.RpcQueryNode;
+import uk.oczadly.karl.jnano.rpc.request.node.RequestBlockCount;
+import uk.oczadly.karl.jnano.rpc.request.node.RequestVersion;
+
+import java.math.BigDecimal;
+import java.net.URI;
+import java.net.URL;
+
+/**
+ * Tests NanoRPCWallet with payments
+ */
+class TestPaymentSupport {
+
+    public static void main(String[] args) {
+        try {
+            // You need to have node running: i.e.: nano_node --daemon with rpc enabled
```

Comment:
guess comment should also mention to recommend having websocket enabled, or is it required?